tests/unit/records/test_receivers.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2016 CERN.
+#
+# INSPIRE is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+from inspirehep.modules.records import receivers
+
+
+def test_populate_recid_from_ref_naming():
+    """Test correct renaming of reference fields."""
+    json_dict = {
+        'simple_key': {'$ref': 'http://x/y/1'},
+        'key_with_record': {'$ref': 'http://x/y/2'},
+        'record': {'$ref': 'http://x/y/3'},
+        'embedded_list': [{'record': {'$ref': 'http://x/y/4'}}],
+        'embedded_record': {'record': {'$ref': 'http://x/y/5'}}
+    }
+    receivers.populate_recid_from_ref(None, json_dict)
```

Comment:
Nitpick, newlines before and after `receivers.populate_recid_from_ref(None, json_dict)`, so it's more clear which are the preconditions, what are you testing, what are your assertions.